packages/strapi-design-system/src/v2/SimpleMenu/SimpleMenu.js

```diff
@@ -38,7 +47,7 @@ const StyledButtonSmall = styled(Button)`
   padding: ${({ theme }) => `${theme.spaces[1]} ${theme.spaces[3]}`};
 `;
 
-export const MenuItem = ({ as, children, onClick, isFocused, isLink, ...props }) => {
+export const MenuItem = ({ as, children, onClick, isFocused, to, href, isExternal, ...props }) => {
```

Comment:
Some comments on `SimpleMenu` v2:
- same padding fix as for SimpleMenu v1 which doesn't take the icon into account
```js
const OptionExternalLink = styled(Link)`
  padding: ${({ theme }) => theme.spaces[2]};
  ...
`;

...

<OptionExternalLink href={href} isExternal={isExternal} {...menuItemProps}>
  <Typography>{children}</Typography>
</OptionExternalLink>
```
- same fix as for `SimpleMenu` v1, recover focusable element
```js
  const menuItemProps = {
    tabIndex: isFocused ? 0 : -1,
    ref: menuItemRef,  // <=== we need the ref
    role: 'menuitem',
    ...props,
  };
```